src/cursor/cur_std.c

```diff
@@ -567,6 +579,42 @@ __wt_cursor_get_valuev(WT_CURSOR *cursor, va_list ap)
     API_END_RET_STAT(session, ret, cursor_get_value);
 }
 
+/*
+ * __wt_cursor_get_raw_key_value --
+ *     WT_CURSOR->get_raw_key_value default implementation
+ */
+int
+__wt_cursor_get_raw_key_value(WT_CURSOR *cursor, WT_ITEM *key, WT_ITEM *value)
+{
+    WT_DECL_RET;
+    WT_SESSION_IMPL *session;
+
+    CURSOR_API_CALL(cursor, session, get_value, NULL);
+
+    if ((key != NULL) && !F_ISSET(cursor, WT_CURSTD_KEY_SET))
+        WT_ERR(__wt_cursor_kv_not_set(cursor, true));
+
+    if ((value != NULL) && !F_ISSET(cursor, WT_CURSTD_KEY_SET))
```

Comment:
I think this should be WT_CURSTD_VALUE_SET

Reply:
Hi Alex, thanks for re-re-reviewing, and well spotted! I've fixed that in commit https://github.com/wiredtiger/wiredtiger/pull/8584/commits/1b65aa97d27e2f21ff9d1f6d8bd4a5bde8153bbf. Thanks, Jeremy